### PR TITLE
Fixes #10 - Adds a link to the original bugzilla issue, sets bugzilla summary as bug description 

### LINKED ISSUE #10: Save a link to the original bugzilla issue when the issue is created on webcompat side
We need to add a link to the original bugzilla issue, so we don't have to create a new bug in case diagnosis resolves in one (can just reopen the original bug).

### PR DESCRIPTION
**Pull request by ksy36**
r? @denschub

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -9,5 +9,6 @@
   },
   "extends": [
     "eslint:recommended"
-  ]
+  ],
+  "ignorePatterns": ["jest.config.js", "**/test/*.test.js", "module_shim.js"]
```
Comment (skyschub) [inline .eslintrc.json]: What's the motivation for excluding test files here? I don't mind and I'm sure you have a reason, but I feel like I could learn something! :)

**Review by skyschub — APPROVED** [review]
Looking good!

File: .eslintrc.json (hunk 1 above)
Comment (ksy36) [reply .eslintrc.json]: I was getting no-undef errors for jest globals (describe, it, etc.). 
Now thinking back, I could probably exclude them with `/* globals .... */ ` comment and do not ignore the test file :)

Comment (ksy36) [reply .eslintrc.json]: Added them to "globals" in eslint config instead

**Comment by ksy36** [comment]
Thanks!

File: .eslintrc.json (hunk 1 above)
Comment (skyschub) [reply .eslintrc.json]: Ah, cool. :) And thanks for the explanation!

**Pull request merged**